app/views/facility_accounts_reconciliation/_table.html.haml

```diff
@@ -1,23 +1,24 @@
 %table.table.table-striped.table-hover
+  %caption= "#{account_class.model_name.human(count: 2)} #{t("views.facility_accounts_reconciliation.index.caption")}"
   %thead
     %tr
       %th
-      %th= Statement.human_attribute_name(:invoice_number)
-      %th= Account.model_name.human
-      %th= Order.model_name.human
-      %th= Statement.human_attribute_name(:total_cost)
-      %th= Statement.human_attribute_name(:reconcile_note)
+      %th{scope: "col"}= Statement.human_attribute_name(:invoice_number)
+      %th{scope: "col"}= Account.model_name.human
+      %th{scope: "col"}= Order.model_name.human
+      %th{scope: "col"}= Statement.human_attribute_name(:total_cost)
+      %th#reconcile-note.row-note-input{scope: "col"}= Statement.human_attribute_name(:reconcile_note)
   %tbody
     - @unreconciled_details.each do |order_detail|
       %tr
-        %td.centered= check_box_tag "order_detail[#{order_detail.id}][reconciled]", "1", false, class: "toggle"
+        %td.centered= check_box_tag "order_detail[#{order_detail.id}][reconciled]", "1", false, class: "toggle", aria: {label: t("views.facility_accounts_reconciliation.index.checkbox-label", order: order_detail.order_id, order_detail: order_detail.id)}
```

Comment:
You can actually just pass in the order_detail here, it has a `to_s` method that will do the same thing:
```suggestion
        %td.centered= check_box_tag "order_detail[#{order_detail.id}][reconciled]", "1", false, class: "toggle", aria: {label: t("views.facility_accounts_reconciliation.index.checkbox-label", order_number: order_detail.to_s)}
```